coordinator/src/test/resources/coordinator.conf

```diff
@@ -28,3 +28,4 @@ rss.coordinator.access.candidates.updateIntervalSec 1
 rss.coordinator.access.loadChecker.serverNum.threshold 2
 rss.coordinator.access.loadChecker.memory.percentage 20.0
 rss.coordinator.dynamicClientConf.updateIntervalSec 1
+plugin.custom.key v1
```

Comment:
Could we have a better format? We will have many plugins. What's type of plugin? The plugin is used for shuffle server , Coordinator or both. I can't get enough information.

Reply:
> Could we have a better format? We will have many plugins. What's type of plugin? The plugin is used for shuffle server , Coordinator or both. I can't get enough information.

It is just for UT and it is not actually used.